src/styles/Book.css

```diff
@@ -0,0 +1,288 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.list-container {
+  width: 75rem;
+  height: 10.625rem;
+  margin: 0 auto;
+  margin-top: 2.313rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.list-block,
+.right-section,
+.list-actions,
+.progress {
+  display: flex;
+}
+
+.progress {
+  flex-direction: column;
+  align-items: center;
+}
+
+.progress-sign {
+  float: left;
+  margin-right: 50px;
+  width: 70px;
+  height: 70px;
+  font-size: 30px;
+  color: #fff;
+  border-radius: 50%;
+  overflow: hidden;
+  position: relative;
+  background: #fff;
+  text-align: center;
+  line-height: 200px;
+}
+
+.progress-sign::after {
+  content: "%";
+}
+
+.progress-sign .title {
+  position: relative;
+  z-index: 100;
+}
+
+.progress-sign .overlay {
+  width: 50%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: #ffff;
+}
+
+.progress-sign .left,
+.progress-sign .right {
+  width: 50%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 5px solid #379cf6;
+  border-radius: 100px 0 0 100px;
+  border-right: 0;
+  transform-origin: right;
+}
+
+.progress-sign .left {
+  animation: load1 1s linear forwards;
+}
+
+.progress-sign:nth-of-type(2) .right,
+.progress-sign:nth-of-type(3) .right {
+  animation: load2 0.5s linear forwards 1s;
+}
+
+.progress-sign:last-of-type .right,
+.progress-sign:first-of-type .right {
+  animation: load3 0.8s linear forwards 1s;
+}
+
+@keyframes load1 {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(180deg);
+  }
+}
+
+@keyframes load2 {
+  0% {
+    z-index: 100;
+    transform: rotate(180deg);
+  }
+
+  100% {
+    z-index: 100;
+    transform: rotate(270deg);
+  }
+}
+
+@keyframes load3 {
+  0% {
+    z-index: 100;
+    transform: rotate(180deg);
+  }
+
+  100% {
+    z-index: 100;
+    transform: rotate(315deg);
+  }
+}
+
+.progress-percent {
+  width: 4.25rem;
```

Comment:
- [ ] kindly shift the add book button a little to the right to it aligns with the book cards as well like in the the design. nice work.
yours
![Screenshot_19](https://github.com/anyars-encarta/book-store/assets/65298048/4ecf28fd-3678-43ac-ab79-a90973390edd)
original
![Screenshot_18](https://github.com/anyars-encarta/book-store/assets/65298048/f512c720-f0a9-45fc-ab15-265e69ab9632)

Reply:
I must say that I really admire your attention to detail. Correcting that asap!